docs/manual/common/concepts/ComponentTechnologies.md

```diff
@@ -0,0 +1,21 @@
+# Component technologies
+
+As a complete microservices platform, Lagom assembles a collection of technologies and adds value on top of them. Some of the libraries, tools, and servers that Lagom uses and supports were developed at [Lightbend](https://lightbend.com), others are third-party and open-source. As you develop with Lagom, you can also take advantage of these technologies:
+
+* Akka --- Lagom [[Persistence|PersistentEntity]], [[Publish-Subscribe|PubSub]], and [[Cluster|Cluster]] are implemented on top of [Akka](http://akka.io/), Lightbend's toolkit for building concurrent, distributed, and resilient message-driven applications. This is an implementation detail that does not directly concern you when developing simple microservices. However, advanced users can call some Akka APIs directly. Specifically:
```

Comment:
Well, the re-write reflects what I understood from the original and your comments, which apparently still isn't correct. I found the original page confusing and thought it would be helped by grouping the Akka technologies together. Since I've heard from several places the importance of highlighting other Lightbend technologies that Lagom relies on, and I think it is important to introduce clustering and streaming in several places in the concepts, I'd prefer not to remove them, But at this point, I'm not sure what to do. I do not feel that the sentence at the end of the first bullet implies that you can call the sub-bullet components directly, that's not what it says. The "For Example" that I put in because you didn't like "Specifically" is misleading, I've taken that out. And, finally, the link to "Advanced Topic:Integrating with Akka" was not removed, the file name is just "Akka".